package.json

```diff
@@ -130,6 +130,20 @@
     "workbox-webpack-plugin": "^6.5.4",
     "yaml-loader": "^0.8.0"
   },
+  "overrides": {
+    "appium-safari-driver": {
+      "archiver-utils": {
+        "glob": {
+          "foreground-child": {
+            "cross-spawn": "^7.0.5"
+          }
+        }
+      }
+    },
+    "express": {
+      "path-to-regexp": "0.1.12"
```

Comment:
Express is part of the test credential server, not here. Not too important if there's vulnerabilities there but I guess better to not have dependabot notifications.

Reply:
Based on the docs, this rule applies to all `path-to-regexp`(any level) after `express`.

In root directory, run:

`npm list path-to-regexp`

Paths related to `path-to-regexp@0.1.10`
```bash
...
├─┬ appium-uiautomator2-driver@3.9.2
│ └─┬ appium@2.13.1
│   └─┬ @appium/base-driver@9.13.1
│     ├─┬ express@4.21.1 overridden
│     │ └── path-to-regexp@0.1.10 overridden
│     └── path-to-regexp@8.2.0
├─┬ appium-xcuitest-driver@7.33.0
│ └─┬ appium-remote-debugger@12.1.4
│   └─┬ @appium/base-driver@9.13.1
│     ├─┬ express@4.21.1 overridden
│     │ └── path-to-regexp@0.1.10 overridden
│     └── path-to-regexp@8.2.0
...
```

Now that I see it better, there it indicates that they are overriden, but I don't know why it is still with version `0.1.10`.

Reply:
Ah appium has express. Looking at that it seems like it's both a dependency of `express` and `@appium/base-driver` so you would need another override.